PACKAGING.md

```diff
@@ -4,36 +4,45 @@ Agama packages are available in the [YaST:Head:Agama project in
 OBS](https://build.opensuse.org/project/show/YaST:Head:Agama). This document summarizes the
 process we follow to build those packages.
 
-The process to build the Ruby-based ones (`rubygem-agama` and `rubygem-agama-cli`) is
-different from the one we use to build the web UI (`cockpit-agama`). The former packages are
-built in the same way that other YaST packages (through Rake tasks), while the latter package is
-automatically built in the Open Build Service.
+The process to build each package is slightly different depending on the technology we are using.
+While the Ruby-based one (`rubygem-agama`) is built as any other YaST package, the web UI
+(`cockpit-agama`) and the CLI (`agama-cli`) rely on [OBS source
+services](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html).
 
-## Releasing a new version
+## Versioning policy
+
+We have decided to follow a single number schema: 1, 2, 3, etc. However, if we need to release a
+hot-fix, we might use a dotted version (e.g., 2.1). Moreover, all the components share the same
+version number. Releasing a new version implies that all of them get the new number, no matter if
+they contain changes or not.
+
+## Bumping the version
 
 In order to release a new version, we need to:
 
-* Update the version number in the `VERSION` file in the corresponding subdirectory. These files are
-  read by the Ruby-based packages when building the gems.
-* Tag the repository with the proper number. The process to build `cockpit-agama` uses this
-  information to infer the version. You can set the tag with something like:
+1. Update the version number in the `service/VERSION` file with the new number. These file is read
+  when building the `rubygem-agama` package.
+2. Run `bundle install` from the `service` directory.
+3. Add entries in the changes files.
+4. Open a pull request to get these changes into the repository.
+5. Once the pull request is merged, tag the repository with the proper version number. The processes
+   to build `cockpit-agama` and `agama-cli` use this information to infer the version. You can set
+   the tag with something like:
 
-      git tag --sign 0.5 --message "Version 0.5"
+      git tag --sign 1 --message "Version 1"
```

Comment:
JFYI:

This looks a lot like [the instructions I have for ruby-dbus](https://github.com/mvidner/ruby-dbus/blob/c7677b911e207405e50e73fa3e93170bbf29fe8d/HOWTO-RELEASE), and I've found that I want to automate it as much as possible, which for me means writing lines that  I can paste into the terminal. Example:

```
Edit VERSION NEWS.md package/rubygem-ruby-dbus.spec

    osc vc package
    git commit -m "Releasing version $(cat VERSION)."
    rake check:changelog # implies :package
    gem2rpm --template opensuse --config package/gem2rpm.yml \
      package/ruby-dbus-$(cat VERSION).gem > package/rubygem-ruby-dbus.spec
    git commit -a --amend
    rake osc:build

Push the versioned commit; tag

    cat VERSION && git tag v$(cat VERSION)
    git push
    git push --tags
```